products/BellHybrid/apps/common/include/common/ExcludedLanguages.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <vector>
+#include <i18n/i18n.hpp>
+
+namespace app
+{
+    const inline std::vector<Language> getExcludedLanguages()
+    {
+        return {{"Svenska"}};
```

Comment:
Why it's excluded?

Reply:
That was the design team's decision. As far as I know, there is no full Svenska translation ready.

Reply:
Confirmed :+1: